python/paddle/fluid/framework.py

```diff
@@ -949,10 +963,10 @@ def _var_recursive(self, name):
         raise ValueError("Var {0} is not found recursively".format(name))
 
     def all_parameters(self):
-        return list(self._iter_parameters())
+        return list(self.iter_parameters())
 
-    def _iter_parameters(self):
-        return (item[1] for item in self.vars.iteritems()
+    def iter_parameters(self):
```

Comment:
-> _iter_parameters

Reply:
Thanks, I will change it later

Reply:
Mabye I'm not... ```iter_parameter``` was used in some CE models, so this will be changed util all related CE models change it's code